pkg/genlib/generator_test.go

```diff
@@ -54,81 +229,111 @@ func Test_Cardinality(t *testing.T) {
 }
 
 func test_CardinalityT[T any](t *testing.T, ty string) {
-
+	template := `{"alpha":"{{.alpha}}"}`
```

Comment:
all these tests do not to seem to be updated to cover `Gen2` and `Gen3` (unless I missed something), we should do :)